src/posix/platform/CMakeLists.txt

```diff
@@ -68,4 +68,5 @@ target_include_directories(openthread-posix PRIVATE
     ${PROJECT_SOURCE_DIR}/src
     ${PROJECT_SOURCE_DIR}/src/core
     ${PROJECT_SOURCE_DIR}/src/posix/platform
+    ${PROJECT_SOURCE_DIR}/third_party/mbedtls/repo/include
```

Comment:
Is this necessary? Is this not getting included as part of `OT_PRIVATE_INCLUDES`?

https://github.com/openthread/openthread/blob/73e30d1d7b1454e0d82607381b6b9c1f79ff4cec/CMakeLists.txt#L91

Reply:
I think it's because this is not included by the platform library.(see the test failure: https://travis-ci.org/openthread/openthread/jobs/634169345).

Reply:
@jwhui the platform library is added before that path is added into `OT_PRIVATE_INCLUDES`.